package.json

```diff
@@ -25,6 +25,7 @@
     "babel": "^5.5.8",
     "babel-core": "^5.2.17",
     "babel-loader": "^5.0.0",
+    "prop-types": "^15.5.8",
```

Comment:
For same reason as `react` is on `peerDependencies`. 

But I was checking here https://github.com/reactjs/prop-types/issues/4#issuecomment-292779211 and maybe on `dependencies` should be also fine.